buildpipeline/pipeline.json

```diff
@@ -45,6 +45,23 @@
             "Type": "build/product/",
             "ConfigurationGroup": "Release"
           }
+        },
+        {
+          "Name": "DotNet-CoreFx-Trusted-Linux-RedHat6",
```

Comment:
I think this name needs to match the build definition name so it should be "Dotnet-CoreFx-Trusted-Linux", same with the other one.